src/modules/claim/gift/GiftDetails.tsx

```diff
@@ -0,0 +1,29 @@
+import { Flex, Hide, Text } from '@chakra-ui/react';
+
+import { AvatarImage } from '@/common/components/AvatarImage';
+
+interface GiftDetailsProps {
+  imageSrc: string;
+  giftName: string;
+}
+
+export const GiftDetails = ({ imageSrc, giftName }: GiftDetailsProps) => {
+  return (
+    <Flex align="center" flexDir="column">
+      <Hide above="md">
+        <Text color="gray.800" fontSize="lg" fontWeight="500" lineHeight="28px" mb="8">
+          Attendance gifts
+        </Text>
+      </Hide>
+      <AvatarImage altName={giftName} imageSrc={imageSrc} />
```

Comment:
FYI, attendance gift can also be tokens,
so this page should display either Tokens or NFT based on backend data
<img width="1389" alt="image" src="https://user-images.githubusercontent.com/19740800/212282190-6ac24837-3a55-4783-9b37-8ac5cba2e132.png">

Reply:
yep yep, the code is already there! right now it's only based on the `[id]` value: odd for NFT, even for token for mocking purposes

Reply:
![image](https://user-images.githubusercontent.com/40631483/212562721-1f01665f-67b6-45f5-8317-e5f5495e342c.png)

![image](https://user-images.githubusercontent.com/40631483/212562732-f14c5afd-92a7-4e00-a852-310ae7d47789.png)